2024/index.html

```diff
@@ -1229,4 +1229,4 @@ <h2 words-slide-from-right="" text-split="" class="white-type">Learn More</h2>
   });
 </script>
 </body>
-</html>
\ No newline at end of file
+</html>

```

Comment:
I assume this was `vim`'s doing? I don't know how we feel about blank lines at the end of HTML files. I have zero opinion.

Reply:
It would've been from whatever the tool their designer used. I didn't check when I copied from the zip file.

Reply:
Hm, maybe I misread the PR. It looked like I removed a blank line. Whatevs.